### ISSUE TITLE
Unthresholded recurrence plot

### ISSUE THREAD

**Issue by ogreyesp** [johannfaouzi/pyts#79]
Does **pyts** library support the generation of unthresholded recurrence plots?

**Comment by johannfaouzi** [comment]
Yes. The corresponding value for the `threshold` parameter is `None` (it's the default value).

See the [documentation of pyts.image.RecurrencePlot](https://pyts.readthedocs.io/en/stable/generated/pyts.image.RecurrencePlot.html#pyts.image.RecurrencePlot).
